newrelic/resource_newrelic_synthetics_multilocation_alert_condition.go

```diff
@@ -81,9 +81,18 @@ func resourceNewRelicSyntheticsMultiLocationAlertCondition() *schema.Resource {
 			},
 			"violation_time_limit_seconds": {
 				Type:         schema.TypeInt,
-				Required:     true,
-				ValidateFunc: validation.IntInSlice([]int{0, 3600, 7200, 14400, 28800, 43200, 86400}),
-				Description:  "The maximum number of seconds an incident can remain open before being closed by the system.  Must be one of: 0, 3600, 7200, 14400, 28800, 43200, 86400",
+				Optional:     true,
+				ValidateFunc: validation.IntBetween(0, violationTimeLimitSecondsMax),
```

Comment:
This is supposed to be 300 - 2592000 as also specified in New Relic docs, but I've added 0 as the minimum as this resource is already in use with 0 as a valid value of `violation_time_limit_seconds`.

However, the current behaviour of the API is such that if `0` is given as a value against this argument, the API defaults it to 259200 seconds. This is why the below `DiffSuppressFunc` has been added.